tests/__init__.py

```diff
@@ -1,2 +1,2 @@
 
-__all__ = ['test_bs', 'misc', 'fixtures']
+__all__ = ['test_bs', 'test_p2p', 'misc', 'fixtures']
```

Comment:
why do we need this for p2p but not for hare, sync, etc.. ?

Reply:
Basically we don't need it at this stage. We might need it in the future.
Removing it for now...